### DISCUSSION TITLE
Build Failed

### DISCUSSION THREAD

**Discussion by iniacor** [vercel/community#2945] [Help] [answered]
### Summary

I got these errors during deploy : 

Error: Function Runtimes must have a valid version, for example `now-php@1.0.0`.
--
19:48:35.722 | invalid_function_runtime: Function Runtimes must have a valid version, for example `now-php@1.0.0`.

But site works and everything OK and i don't use php in this implementation just node. This is my project https://powercode-experiment.vercel.app/ and this is my git  repo https://github.com/iniacor/powercode-experiment

### Steps to Reproduce

redeploy project on Vercel

**Comment by amyegan** [comment, marked as answer]
The `vercel.json` config is not needed. If you remove it, the error will go away 🙂

**Reply by iniacor** [reply]
@amyegan Hi, i removed it. But now site does not work. Just 404 error on display

**Comment by iniacor** [comment]
@amyegan  Hi, i removed it. But now site does not work. Just 404 error on display
